Sources/Onboarding/Callouts/OnboardingLinkReceiptView.swift

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Shared
+import BraveShared
+import DesignSystem
+import BraveUI
+
+public struct OnboardingLinkReceiptView: View {
+  @Environment(\.presentationMode) @Binding private var presentationMode
+  
+  public var linkReceiptAction: (() -> Void)?
+  
+  public init() {}
```

Comment:
nit: dont think empty constructor is necessary

Reply:
yes its separate module, i have to add it